indico/modules/events/management/client/js/index.js

```diff
@@ -35,6 +36,22 @@ import {SingleEventMove, EventPublish} from './EventMove';
       );
     }
 
+    const manageSeriesContainer = document.querySelector('#event-action-manage-series-container');
+    if (manageSeriesContainer) {
+      const hasSeries = manageSeriesContainer.dataset.seriesId !== undefined;
+      const seriesId = hasSeries ? +manageSeriesContainer.dataset.seriesId : null;
+      ReactDOM.render(
+        React.createElement(SeriesManagement, {
+          eventId: +manageSeriesContainer.dataset.eventId,
+          categoryId: +manageSeriesContainer.dataset.categoryId,
+          timezone: manageSeriesContainer.dataset.timezone,
+          hasSeries,
+          seriesId,
```

Comment:
I'd still get rid of `hasSeries` here (and move it into the component) - better to have just one source of truth (`seriesId !== null` (or undefined))

Reply:
oh lol I misunderstood what you meant. I mean, I got rid of it in the step before, but didn't got that far